Makefile

```diff
@@ -3,4 +3,23 @@ run:
 run-debug:
 	cargo run -- --dev --tmp --enable-offchain-indexing=true -l sc_offchain=trace
 keystore-add:
-	curl http://localhost:9933 -H "Content-Type:application/json;charset=utf-8" -d "@scripts/keystore.json"
\ No newline at end of file
+	curl http://localhost:9933 -H "Content-Type:application/json;charset=utf-8" -d "@scripts/keystore.json"
+
+code-prettier:
+	cargo fmt
+
+benchmarks-build:
```

Comment:
In the future, is there some way we can run these in CI to see if pallet changes in PR perform to a certain benchmark?

Reply:
Yes this should be a part od our CI steps. We can simply run those two commands to get it done:
- `make benchmarks-build`
- `make benchmarks-generate`
- then regular substrate **build command**

Otherwise we may want to do it on local dev only when a new extrinsic benchmarks are added

Reply:
cc @aelesbao